pom.xml

```diff
@@ -326,6 +326,28 @@
                 <artifactId>httpcore</artifactId>
                 <version>${httpcore.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Can you please fix the order of dependencies ? The dependencies in this POM is separated into different sections. Please do so for the dependencies added in the last PR